script/Arcueid/Day 11/11_00_ARC11_1.txt

```diff
@@ -347,7 +347,7 @@ Arihiko shrugs his shoulders exaggeratedly.
 [sha:2c0e2f8db8f6084b1d5fcd0daeffae218d0ad769]{
 -- Page 23, Offset 29539. Mods: @n.
 -- 「……なに言ってるんだおまえ。先輩は―――
-"...What do you mean you don't? I'm talking about―――
+"...What do you mean you don't? Senpai is―――
```

Comment:
I'm assuming this sentence in full is along the lines of "I'm talking about Ciel-senpai here", hence the non-literal TL here. Just "senpai is" doesn't really make as much sense as a fragment.